### PR TITLE
Added fix for long names using host aliases.

### PR DESCRIPTION
**Pull request by LamaAni**
## What:
Changed the inner container host resolve for postgres and nc using HOSTALIASES enviroment variable. The host names are now aliased by,
```
local-alias-[type]-[#]
```
For example, for a datanode: 
```
local-alias-dn-0
```

The aliases are loaded into the config directory @ `/config/host_aliases` and a env is added named `HOSTALIASES=/config/host_aliases` to match.

Then, create node uses these aliases to connect to the other nodes on the network.

## Why:
Fix for issue: https://github.com/LamaAni/postgres-xl-helm/issues/36

### CODE REVIEW

**Comment by stephenstubbs** [comment]
Brilliant! Will test this now.

**Comment by LamaAni** [comment]
> Brilliant! Will test this now.

Awesome, waiting on your approval.

**Review by stephenstubbs — APPROVED** [review]
Sorry it took me so long on this. Was trying to get it tested with the test script I'm working on but no point holding this up any longer. Its working well for me.

**Pull request merged**
